pkg/cluster/cluster.go

```diff
@@ -133,6 +134,10 @@ func new(config Config, e *spec.EtcdCluster, stopC <-chan struct{}, wg *sync.Wai
 		if err := c.createClientServiceLB(); err != nil {
 			return nil, fmt.Errorf("fail to create client service LB: %v", err)
 		}
+	} else {
+		if !c.bm.isBackupSetup() {
+			return nil, errors.New("")
```

Comment:
> it is still not clear to me that why do we need to check backup service if it is not a new cluster.

When we new() a cluster, if it has backup policy and thus has a backup manager, but behind it the backup service doesn't exist, i.e. we couldn't get backup or save backup, it doesn't make sense have such a cluster.